config.gradle

```diff
@@ -11,34 +11,34 @@
  * and fluentui_drawer' current version is 0.0.2, both fluentui_listitem and
  * fluentui_drawer and FluentUI should increment their respective version ids
   */
-project.ext.fluentui_calendar_versionid='0.0.21'
-project.ext.fluentui_core_versionid='1.0.0'
-project.ext.fluentui_listitem_versionid='0.0.21'
-project.ext.fluentui_tablayout_versionid='0.0.21'
-project.ext.fluentui_drawer_versionid='0.0.21'
-project.ext.fluentui_ccb_versionid='0.0.21'
-project.ext.fluentui_others_versionid='1.0.0'
-project.ext.fluentui_transients_versionid='0.0.21'
-project.ext.fluentui_topappbars_versionid='0.0.21'
-project.ext.fluentui_menus_versionid='0.0.21'
-project.ext.fluentui_peoplepicker_versionid='0.0.21'
-project.ext.fluentui_persona_versionid='0.0.21'
-project.ext.fluentui_progress_versionid='0.0.21'
-project.ext.FluentUI_versionid='1.0.0'
-project.ext.fluentui_calendar_version_code=21
-project.ext.fluentui_core_version_code=22
-project.ext.fluentui_listitem_version_code=21
-project.ext.fluentui_tablayout_version_code=21
-project.ext.fluentui_drawer_version_code=21
-project.ext.fluentui_ccb_version_code=21
-project.ext.fluentui_others_version_code=22
-project.ext.fluentui_transients_version_code=21
-project.ext.fluentui_topappbars_version_code=21
-project.ext.fluentui_menus_version_code=21
-project.ext.fluentui_peoplepicker_version_code=21
-project.ext.fluentui_persona_version_code=21
-project.ext.fluentui_progress_version_code=21
-project.ext.FluentUI_version_code=34
+project.ext.fluentui_calendar_versionid='0.0.23'
+project.ext.fluentui_core_versionid='0.1.0'
+project.ext.fluentui_listitem_versionid='0.0.23'
+project.ext.fluentui_tablayout_versionid='0.0.23'
+project.ext.fluentui_drawer_versionid='0.0.23'
+project.ext.fluentui_ccb_versionid='0.0.23'
+project.ext.fluentui_others_versionid='0.1.0'
+project.ext.fluentui_transients_versionid='0.0.23'
+project.ext.fluentui_topappbars_versionid='0.0.23'
+project.ext.fluentui_menus_versionid='0.0.23'
+project.ext.fluentui_peoplepicker_versionid='0.0.23'
+project.ext.fluentui_persona_versionid='0.0.23'
+project.ext.fluentui_progress_versionid='0.0.23'
+project.ext.FluentUI_versionid='0.1.0'
+project.ext.fluentui_calendar_version_code=23
+project.ext.fluentui_core_version_code=23
```

Comment:
This is the very same release with correct name. I don't think version code needs changing